Dockerfile

```diff
@@ -1,4 +1,4 @@
-#  Copyright (C) 2018-2021 LEIDOS.
```

Comment:
This should not have been deleted. Please add back

Reply:
Added line back in dockerfile